embedded-memsql/src/main/java/com/playtika/test/memsql/EmbeddedMemSqlTestOperationsAutoConfiguration.java

```diff
@@ -38,14 +33,16 @@ public PackageInstaller memsqlPackageInstaller(
             InstallPackageProperties memsqlPackageProperties,
             @Qualifier(BEAN_NAME_EMBEDDED_MEMSQL) GenericContainer memsql
     ) {
-        return new AptGetPackageInstaller(memsqlPackageProperties, memsql);
+        return new YumPackageInstaller(memsqlPackageProperties, memsql);
     }
 
-    @Bean
-    @ConditionalOnMissingBean(name = "memsqlNetworkTestOperations")
-    public NetworkTestOperations memsqlNetworkTestOperations(
-            @Qualifier(BEAN_NAME_EMBEDDED_MEMSQL) GenericContainer memsql
-    ) {
-        return new DefaultNetworkTestOperations(memsql);
-    }
+// Current image doesn't support `tc` command, since kafka is currently based on ubi with microdnf package manager. `iproute2` package is not available here.
```

Comment:
Current image doesn't support `tc` command, since kafka is currently based on ubi with microdnf package manager. `iproute2` package is not available here.?
:)